### ISSUE TITLE
Math preview

### ISSUE THREAD

**Issue by JackGin** [sublimelsp/LSP-TexLab#2]
Hi! Thank you for your very much needed plugin! Do you have a quick preview of math environments? Like phantoms in LaTeXTools? The LSP plugin for VSC does have the math preview, too. I do not see them working out of the box in sublimelsp right now.

p.s. and by the way, how do I start the build system? what is the shortcut for the compile document? F5 does not work for me.

**Comment by LDAP** [comment]
Hi! We provide a bridge to the TexLab language server. In the most recent release there were some changes: See https://github.com/latex-lsp/texlab/blob/master/CHANGELOG.md

- The math preview was disabled. 
- Building with _save_ is not supported anymore for some reason.  We provide custom commands to compile manually. Have a look in our [README](https://github.com/sublimelsp/LSP-TexLab/blob/master/README.md)

It is possible to integrate TexLab with the BuildSystem of Sublime Text by creating two files in the `User` folder:
Create a file `LSP-TexLab.sublime-build` with 
```json
{
    "target": "lsp_texlab_build_system",
    "selector": "text.tex.latex"
}
```
And a `lsp_textlab_build.py` with
```python
import sublime_plugin
class LspTexlabBuildSystemCommand(sublime_plugin.WindowCommand):
    def is_enabled(self, lint=False, integration=False, kill=False) -> bool:
        return True

    def run(self, lint=False, integration=False, kill=False):
        if kill:
            print("TexLab: Killing builds is not supported")
            return
        self.window.active_view().run_command("lsp_texlab_build")
        # comment this line if you do not want forward search
        self.window.active_view().run_command("lsp_texlab_forward_search")
```

**Comment by LDAP** [comment]
@jfcherng maybe we should ship LSP-TexLab with the build system included?

**Comment by jfcherng** [comment]
> @jfcherng maybe we should ship LSP-TexLab with the build system included?

Why not? I don't actually use this plugin. It looks like you are the most proper one to maintain this plugin.

**Comment by linmajia** [comment]
Looking forward to seeing the new release with the build system!
BTW: TexLab has been upgraded to 3.0.1

**Comment by LDAP** [comment]
The build system was included here 2e6f0921648a4ab6534319e121c3eacd52fd1082.
